01-Introduction.Rmd

```diff
@@ -9,11 +9,13 @@ Contributions to base R are possible in a number of different ways. Some of them
 1. Contributing to bug fixing: Refer [Bug Tracking](#BugTrack) and [Reviewing Bugs](#ReviewBugs).
 2. Contributing to translations: Refer [Translations](#).
 3. Testing R before release: Refer [Testing Pre-release R Versions](#TestRVer).
+4. Contributing to documentation and tutorials, including this guide: Refer [Documenting R](#Doc) and [Contributing to this guide](#how-to-contribute-to-this-guide).
 
-<!-- TODO(Saranjeet): Refer Contributing to R tutorial -->
 <!-- TODO(Saranjeet): Welcome to Bug BBQ slides -->
 <!-- TODO(Saranjeet): README on Bug BBQ repo -->
 
+Maintainers and contributors are requested to follow this project's [code of conduct](https://github.com/r-devel/rdevguide/blob/main/CONDUCT.md).
```

Comment:
This change is not directly related to my aim of helping people contribute to docs, but in reading as a new contributor I found I clicked away on links before getting to this important statement at the end of the page, so it may be better to put it higher up to ensure people are aware fo the CoC before they go into the details of how to contribute.